elm_frontend/src/GeneralTypes.elm

```diff
@@ -0,0 +1,58 @@
+module GeneralTypes exposing (DOI, NamedUrl, PaperMetadata, renderPaperMetaData, renderUrl)
```

Comment:
The contents of this seems to be general as in it's not part of any individual paper but also still exclusive to papers, it seems. Also, none of the other files with similar contents (types and views) has types in its name, which might be confusing some into believing that this file only contains types.
How about we make this `Papers.General` or `Papers.Utils` or if we find it to fit on the top level `PaperUtils`?

Reply:
I like `Papers.Utils`